res/lang/bossbetty/bossbetty_en.txt

```diff
@@ -17,62 +17,62 @@ DIALOGUE_BETTY_WELCOME_LEVEL1_TEXT5 = Hi %PLAYERNAME%,\n\nour latest meeting was
 DIALOGUE_BETTY_WELCOME_LEVEL1_TEXT6 = Hey there!\n\nI'll make us some tea - hope you have some time to chat.
 
 DIALOGUE_BETTY_NO_AWARD_LEVEL0_TEXT1                              = Currently, no Sammy is awarded. I think in some days the race for points will start again.
-DIALOGUE_BETTY_NO_AWARD_LEVEL1_TEXT1                              = Currently, no Sammy is awarded. Maybe I can tell you more tomorrow.
-DIALOGUE_BETTY_AWARDINFORMATION_NO_FAVORITE_LEVEL0_TEXT1          = The %AWARDNAME%?\n\nThere is no favorite yet.
-DIALOGUE_BETTY_AWARDINFORMATION_YOU_ARE_FAVORITE_LEVEL0_TEXT1     = The %AWARDNAME%?\n\nCurrently, it looks good for you.
-DIALOGUE_BETTY_AWARDINFORMATION_YOU_ARE_FAVORITE_LEVEL1_TEXT1     = The %AWARDNAME%?\n\nCurrently, it looks good for you.
-DIALOGUE_BETTY_AWARDINFORMATION_YOU_ARE_AVERAGE_LEVEL0_TEXT1      = The %AWARDNAME%?\n\nThis I can tell: You are ranking amongst the others.
-DIALOGUE_BETTY_AWARDINFORMATION_YOU_ARE_AVERAGE_LEVEL1_TEXT1      = The %AWARDNAME%?\n\nThis I can tell: You are ranking amongst the others.
+DIALOGUE_BETTY_NO_AWARD_LEVEL1_TEXT1                              = Currently, no Sammy is awarded %PLAYERNAME%. Maybe I can tell you more tomorrow.
```

Comment:
comma before playername? in the following entries as well

Reply:
Probably I thought without the comma it could sound more intimate, but fine, adding them.